app/src/main/java/com/sun/noteapp/ui/home/MainActivity.kt

```diff
@@ -133,6 +164,34 @@ class MainActivity : AppCompatActivity(),
             startActivity(TrashActivity.getIntent(this))
             true
         }
+        R.id.navAllNote -> {
+            presenter.getAllNote()
+            drawerNavigate.closeDrawer(GravityCompat.START)
+            true
+        }
+        R.id.navRemindNote -> {
+            if (!SharePreferencesHelper.isRemind) {
+                SharePreferencesHelper.isRemind = true
+                option = NoteOption(
+                    SharePreferencesHelper.color,
+                    selectedLabels,
+                    SharePreferencesHelper.sortType,
+                    SharePreferencesHelper.isRemind
+                )
+                presenter.getAllNotesWithOption(option)
+            } else {
+                SharePreferencesHelper.isRemind = false
+                option = NoteOption(
+                    SharePreferencesHelper.color,
+                    selectedLabels,
+                    SharePreferencesHelper.sortType,
+                    SharePreferencesHelper.isRemind
+                )
+                presenter.getAllNotesWithOption(option)
```

Comment:
Đoạn này bị lặp code này 
```suggestion
                SharePreferencesHelper.isRemind = !SharePreferencesHelper.isRemind
```
anh cũng có thể dùng run, apply hoặc with cho SharePreferencesHelper cho gọn 
```suggestion
                with (SharePreferencesHelper) {
                    isRemind = !isRemind
                    option = NoteOption(color,  selectedLabels, sortType, isRemind)
                    ....
                )
```